torch/lib/ATen/templates/Tensor.h

```diff
@@ -126,7 +126,7 @@ struct Tensor {
   template<typename T>
   T * data() const;
 
-  void * unsafeGetTH(bool retain) {
+  void * unsafeGetTH(bool retain) const {
```

Comment:
This change is in https://github.com/zdevito/ATen/pull/57. I will rebase on top of that once it lands.